docs/integrations/github/discovery.md

```diff
@@ -153,6 +160,11 @@ This provider supports multiple organizations via unique provider IDs.
   Defaults to `false`.
   Due to limitations in the GitHub API's ability to query for repository objects, this option cannot be used in
   conjunction with wildcards in the `catalogPath`.
+- **`checkRepositoryFiltersForWebhook`** _(optional)_:
+  Whether to validate push events received from webhook.
+  This option enforce check the repository from push event to be matched against the filters configured in the entity provider.
+  Defaults to `false`.
+  It is disabled for default because you can configure the webhook only in expected repositories
 - **`schedule`** _(optional)_:
```

Comment:
as setting up `GithubEntityProvider` changed a bit, in particular for the event-based updates, this should be reflected at this documentation page, too.